apps/docs/src/examples/renderers/CSS2DRenderer/Scene.svelte

```diff
@@ -0,0 +1,59 @@
+<script lang="ts">
+  import { tick } from 'svelte'
+  import { T, useFrame, useRender, useThrelte } from '@threlte/core'
+  import { OrbitControls } from '@threlte/extras'
+  import { CSS2DRenderer } from 'three/addons/renderers/CSS2DRenderer.js'
+  import CssObject from './CssObject.svelte'
+
+  // Set up the CSS2DRenderer to run in a div placed atop the <Canvas>
+  const element = document.querySelector('#css-renderer-target')
+  const cssRenderer = new CSS2DRenderer({ element })
+
+  const { size } = useThrelte()
+  $: cssRenderer.setSize($size.width, $size.height)
+
+  useRender(async (ctx) => {
+    await tick()
+    ctx.renderer?.render(ctx.scene, ctx.camera.current)
+    cssRenderer.render(ctx.scene, ctx.camera.current)
+  })
```

Comment:
Lets destructure the context to `{renderer, scene, camera}` since this is more idiomatic and in line with other examples in the docs.

This could use some comments: why are we doing using `tick()`? What happens after?